app/models/appointment_summary.rb

```diff
@@ -0,0 +1,30 @@
+class AppointmentSummary
+  include ActiveModel::Model
+
+  attr_accessor :supplementary_benefits, :supplementary_debt, :supplementary_ill_health,
```

Comment:
We repeat this list of fields a lot, maybe a constant to hold the list would reduce duplication?